### ISSUE TITLE
[Intro to iOS] Project 1 - Review my app #1

### ISSUE THREAD

**Issue by Designs1008** [Designs1008/-Intro-to-iOS-Project-1#1]
My tip app is complete, please review. /cc @codepathreview

**Comment by timothy1ee** [comment]
Thanks, but if you look at your submission on Github, the only file is the README. There's no gif, and there's no Xcode project. Can you try syncing with Github again?

**Comment by Designs1008** [comment]
![tip_demo6](https://cloud.githubusercontent.com/assets/10554067/5817021/d74dba70-a05b-11e4-9312-885e70238aad.gif)

**Comment by Designs1008** [comment]
Hello Timothy,

I wanted to make sure you received my GitHub updates and comments with my
Licecap gif and my xcode code. I really enjoyed your Intro course and I'm
really hoping I make it in the course.

-Morgan

**Comment by timothy1ee** [comment]
No, it's still not synced. Unfortunately, the class is full now. The next session begins May 18th.
